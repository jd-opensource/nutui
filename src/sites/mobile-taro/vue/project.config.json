{
  "miniprogramRoot": "./dist",
  "projectname": "@nutui/nutui-taro-mobile",
  "description": "nutui-taro-vue",
  "appid": "touristappid",
  "setting": {
    "urlCheck": true,
    "es6": false,
    "postcss": false,
    "minified": false
  },
  "compileType": "miniprogram"
}
