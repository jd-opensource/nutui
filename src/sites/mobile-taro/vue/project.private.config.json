{
  "setting": {},
  "condition": {
    "plugin": {
      "list": []
    },
    "game": {
      "list": []
    },
    "gamePlugin": {
      "list": []
    },
    "miniprogram": {
      "list": [
        {
          "name": "business/pages/address/index",
          "pathName": "business/pages/address/index",
          "query": "",
          "scene": null
        },
        {
          "name": "pages/index/index",
          "pathName": "pages/index/index",
          "query": "",
          "scene": null
        },
        {
          "name": "feedback/pages/noticebar/index",
          "pathName": "feedback/pages/noticebar/index",
          "query": "",
          "scene": null
        },
        {
          "name": "feedback/pages/collapse/index",
          "pathName": "feedback/pages/collapse/index",
          "query": "",
          "scene": null
        },
        {
          "name": "feedback/pages/progress/index",
          "pathName": "feedback/pages/progress/index",
          "query": "",
          "scene": null
        },
        {
          "name": "feedback/pages/swiper/index",
          "pathName": "feedback/pages/swiper/index",
          "query": "",
          "scene": null
        }
      ]
    }
  }
}