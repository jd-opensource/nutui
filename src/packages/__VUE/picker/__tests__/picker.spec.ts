import { config, mount } from '@vue/test-utils';
import Picker from '../index.vue';
import NutIcon from '../../icon/index.vue';
import NutPupup from '../../popup/index.vue';
import NutPickerColumn from '../Column.vue';
import { nextTick, toRefs, reactive, ref, onMounted } from 'vue';

beforeAll(() => {
  config.global.components = {
    NutIcon,
    NutPupup,
    NutPickerColumn
  };
});

afterAll(() => {
  config.global.components = {};
});

const simpleColumn = ['1990', '1991', '1992', '1993', '1994', '1995'];
const columns = [
  {
    values: ['vip', 'normal'],
    className: 'column1'
  },
  {
    values: simpleColumn,
    className: 'column2'
  }
];

test('base', async () => {
  const wrapper = mount(Picker, {
    props: {
      modelVisible: true,
      listData: []
    }
  });
  await nextTick();
});
