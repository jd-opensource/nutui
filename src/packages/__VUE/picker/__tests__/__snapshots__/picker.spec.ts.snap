// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`base 1`] = `
"<view class=\\"nut-picker\\" modelvisible=\\"true\\">
  <!--teleport start-->
  <!--teleport end-->
</view>"
`;
